.rubocop.yml

```
@@ -25,6 +27,161 @@ Layout/TrailingEmptyLines:
 Layout/TrailingWhitespace:
   Enabled: true
 
+Performance/AncestorsInclude:
```

We just need to the root config and we're done.

```suggestion
Performance:
  Enabled: true

Performance/AncestorsInclude:
```